crates/stdarch-verify/tests/x86-intel.rs

```diff
@@ -284,6 +273,7 @@ fn verify_all_signatures() {
                 "_mm_cvtsi64x_sd",
                 "_bextr2_u64",
                 "_mm_tzcnt_64",
+                "_mm_broadcastsi128_si256",
```

Comment:
Can you write a quick test for this to avoid needing to add yet another exception to this list?

Reply:
I think it should be on that list - as that is just a list of all aliases. But ok, I will write a test